documentation/secure_storage.md

```diff
@@ -10,14 +10,16 @@ integrity of the data stored and the atomicity of the operations that modifies
 the storage (atomicity here means that either the entire operation completes
 successfully or no write is done).
 
-There are currently two secure storage implementations in OP-TEE:
+There are currently three secure storage implementations in OP-TEE:
 
 - The first one relies on the normal world (REE) file system. It is described in
 this document and is the default implementation. It is enabled at compile time
 by CFG_REE_FS=y.
 - The second one makes use of the Replay Protected Memory Block (RPMB) partition
 of an eMMC device, and is enabled by setting `CFG_RPMB_FS=y`. It is described
 in [secure_storage_rpmb.md](secure_storage_rpmb.md).
+- The third one stores objects in a SQLite database in normal world. It is
+enabled by `CFG_SQL_FS=y`. See [secure_storage_sql.md](secure_storage_sql.db).
 
 It is possible to use the normal world filesystem and the RPMB implementations
 simultaneously. For this, two OP-TEE specific storage identifiers have been
```

Comment:
Yes it is possible, we do have 3 identifiers, I just need to rephrase the text that follows my addition. Good catch, will fix.